de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/AbstractOutgoingReferencePointNegotiation.java

```diff
@@ -56,10 +57,10 @@
 
     private PacketCollector startActivityQueuingResponseCollector;
 
-    protected AbstractOutgoingProjectNegotiation( //
+    protected AbstractOutgoingReferencePointNegotiation( //
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 11 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=259&issues=AWZfKMfJDZywvPe4LUfh&open=AWZfKMfJDZywvPe4LUfh)

[ISSUE_KEY]: AWZfKMfJDZywvPe4LUfh